docs/docs/workflows/settings/README.md

```diff
@@ -23,7 +23,7 @@ You can disable these notifications for your workflow by disabling the **Notify
 
 <img src="https://res.cloudinary.com/pipedreamin/image/upload/v1656631849/docs/Screen_Shot_2022-06-30_at_4.30.44_PM_oauty4.png" width="200px" alt="Notify me on errors toggle">
 
-## Error Reruns
+## Auto-retry Errors
```

Comment:
```suggestion
## Auto-retry errors
```

Reply:
Changed "Errors" to lowercase "errors" to follow Google's style guide on capitalization.